desktop/release-notes.md

```diff
@@ -24,12 +24,53 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## Docker Desktop 4.11.1
+2022-08-05
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### Bug fixes and minor changes
+
+#### For all platforms
+
+- Fixed `docker login` to private registries when Registry Access Management is enabled and access to Docker Hub is blocked [docker/for-win#12871](https://github.com/docker/for-win/issues/12871)
```

Comment:
Lots of people on the original ticket claimed that they had the issue even when RAM wasn't enabled. In fact, several of them weren't even on Business subscriptions. So (1) are we sure it's fixed in that case? (2) if so, the release note doesn't accurately describe the impact.